uiowa/uiowa
# Hotfix: render uiowa_hours via JS.

Related to #4651 

Hotfix to get caching old hours resolved ASAP. The ideal scenario is probably what I described in #4651 as setting up a new route, controller, services, etc. and then making the AJAX call directly in JS instead of autosubmitting the form. 

# How to test


- `blt sync --site recserv.uiowa.edu`
- See the update hook runs.
- Visit building pages and see hours loaded via JS.
- Log in and verify that the hours form is not submitted while editing the page in Layout Builder and that placeholder text is visible instead.
- Double check the https://recserv.uiowa.ddev.site/hours page works correctly.

Also pushed to dev for Varnish testing: https://recserv.dev.drupal.uiowa.edu
